plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -1806,7 +1811,10 @@ namespace {
 // track multiple failures on unapplied transactions
 class account_failures {
 public:
-   constexpr static uint32_t max_failures_per_account = 3;
+  explicit account_failures(uint32_t max_failures)
+  :max_failures_per_account(max_failures)
```

Comment:
Not sure how this even compiles for you. max_failures_per_account needs to be a member of `account_failures` not `account_failure`

Reply:
I hadn't had a chance to build it yet. I apologize, it should have been in `draft` to indicate that.